.github/workflows/pr-lint.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           PR_BODY=$(gh pr view ${{ github.event.pull_request.number }} --json body --jq ".body")
-          ISSUES=$(echo $PR_BODY | tr '[:upper:]' '[:lower:]' | sed -n -E '/(close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved|relates to|related to|part of) #[0-9]+/p')
+          ISSUES=$(echo $PR_BODY | tr '[:upper:]' '[:lower:]' | sed -n -E '/((close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved|relates to|related to|part of) (.+[0-9]+))/p'
```

Comment:
Ideally I'd make this more specific `(#|https://github.com/opentffoundation/opentf/issue/)[0-9]+`, as right now it _technically_ can catch the wrong sentences and mark them as a link to an issue

Reply:
a11d6e31(more specific)